src/CHANGES.txt

```diff
@@ -6,6 +6,9 @@
 
 
 RELEASE 3.1.0.alpha.yyyymmdd - NEW DATE WILL BE INSERTED HERE
+  From Paweł Tomulik:
```

Comment:
Can you add based on alphabetical last name. That's how we sort these prior to release. So less editing for me is welcome.